### PR TITLE
W-14078975-Partner Manager v2.17 doc edits

### PR DESCRIPTION
**Pull request by dhirsch1**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/partner-manager-identifiers.adoc [hunk 1]
```diff
@@ -87,17 +87,17 @@ Use the following values to configure the X12 GS identifier type:
 [[reference-id]]
 == Outbound Message Flow Partner Reference Identifiers
 
-Outbound message flows that transform your backend application messages to EDI format use the partner Reference ID when sending data to your partners.
+Outbound message flows that transform your backend-application messages to EDI format use the partner Reference ID when sending data to your partners.
```
Comment (valkyrie69) [inline modules/ROOT/pages/partner-manager-identifiers.adoc]: Just want to doublecheck that you do mean backend-application, not back-end application.

File: modules/ROOT/pages/partner-manager-identifiers.adoc [hunk 2]
```diff
@@ -87,17 +87,17 @@ Use the following values to configure the X12 GS identifier type:
...
 
-Unlike EDI transactions that have well-defined standards for how sender, receiver, and document type information is presented in the data, the source messages for outbound B2B data exchange originating from your enterprise backend systems typically follow schemas. These schemas are usually defined by your backend application teams according to your organization’s data models.
+Unlike EDI transactions that have well-defined standards for how sender, receiver, and document type information is presented in the data, the source messages for outbound B2B data exchange originating from your enterprise backend systems typically follow schemas. These schemas are usually defined by your back-end application teams according to your organization’s data models.
```
Comment (valkyrie69) [inline modules/ROOT/pages/partner-manager-identifiers.adoc]:
```suggestion
Unlike EDI transactions that have well-defined standards for how sender, receiver, and document type information is presented in the data, the source messages for outbound B2B data exchange originating from your enterprise back-end systems typically follow schemas. These schemas are usually defined by your back-end application teams according to your organization’s data models.
```

Comment (valkyrie69) [reply modules/ROOT/pages/partner-manager-identifiers.adoc]: Disregard if I'm wrong. I always get confused about when it's backend vs back-end.

File: modules/ROOT/pages/x12-identity-settings.adoc [hunk 3]
```diff
@@ -76,7 +76,7 @@ Refer to this information specifically for using the AS2 identity to identify a
 |Setting |Description |Required
 
 |Identifier Value
-|The AS2 identity for your partner. Your partner should supply you with corresponding to the Sender / Receiver AS2 identity.
+|The AS2 identity for your partner. Your partner should supply you with corresponding to the Sender/Receiver AS2 identity.
```
Comment (valkyrie69) [inline modules/ROOT/pages/x12-identity-settings.adoc]:
```suggestion
|The AS2 identity for your partner. Your partner should supply you with the value of the corresponding Sender/Receiver AS2 identity.
```

Comment (valkyrie69) [reply modules/ROOT/pages/x12-identity-settings.adoc]: ?

**Review by valkyrie69 — DISMISSED** [review]

File: modules/ROOT/pages/partner-manager-identifiers.adoc (hunk 2 above)
Comment (dhirsch1) [reply modules/ROOT/pages/partner-manager-identifiers.adoc]: You are right.

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**